drivers/hmis/lib/form_data/default/fragments/client_demographics.json

```diff
@@ -93,6 +93,7 @@
         }
       ]
     },
+    { "fragment": "#4_21_sex" },
```

Comment:
Move this to a separate branch since we should not start collecting it until FY26